lib/Launcher.js

```diff
@@ -155,17 +150,19 @@ class Launcher {
     /** @type {?Connection} */
     let connection = null;
     try {
-      const connectionDelay = options.slowMo || 0;
+      const {
+        ignoreHTTPSErrors = false,
+        defaultViewport = {width: 800, height: 600},
+        slowMo = 0
+      } = options;
```

Comment:
Will this not cause timeout to be `null` below and always default to 30000?

Reply:
Getting issues with third parties using puppeteer where users are reporting not being able to change timeout anymore.

Reply:
The code for setting the timeout is two lines below this, and remains unchanged. Can you file an issue? I don't think this patch is the cause.

Reply:
@Palgie this code above is simply pulling values OUT of the options object using es6 destructuring . So it is not the cause of the timeout value not being set as you described [here](https://github.com/chrisvfritz/prerender-spa-plugin/issues/222)